src/js/app.js

```diff
@@ -0,0 +1,48 @@
+import { Lotto } from "./lotto.js";
+import { isValidPurchaseAmount } from "./utils.js";
+
+const ERROR_MESSAGE = "1,000원 단위로 입력하세요.";
+const AMOUNT_UNIT = 1000;
```

Comment:
```suggestion
const AMOUNT_UNIT = 1_000;
```
이렇게 표현할 수도 있답니다 ㅎㅎ
지금 당장은 이런 표현을 사용하지 않아도 되지만, 나중에 숫자가 커지는 경우(1_000_000_000_000 처럼)에는 필요할 수 있겠죠!?